### PR TITLE
added tasks related structures_and_classes

### PR DESCRIPTION
**Pull request by Ypostolov**

### CODE REVIEW

File: week_4/invert_func_with_multiplication_division/invert_func_with_multiplication_division.cpp [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+struct Params {
+  double a;
+  double b;
+  double c;
+};
+
+
+class FunctionPart {
+ 
+  private:
```
Comment (ZhdanovP) [inline week_4/invert_func_with_multiplication_division/invert_func_with_multiplication_division.cpp]: @Ypostolov Style problem? 
BTW why are you using private key here?

File: week_4/invert_func_with_multiplication_division/invert_func_with_multiplication_division.cpp [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      operation = '*';
+    }
+  }
+ };
+
+class Function {
+
+private:
+  std::vector<FunctionPart> parts;
+
```
Comment (ZhdanovP) [inline week_4/invert_func_with_multiplication_division/invert_func_with_multiplication_division.cpp:56]: @Ypostolov What is the constructor for Function?

Comment (Ypostolov) [reply week_4/invert_func_with_multiplication_division/invert_func_with_multiplication_division.cpp:56]: Please clarify.

File: week_4/invert_func_with_multiplication_division/invert_func_with_multiplication_division.cpp (hunk 1 above)
Comment (Ypostolov) [reply week_4/invert_func_with_multiplication_division/invert_func_with_multiplication_division.cpp]: Fixed.
Please check 14f1cb1cd14fd68b12659def58de3091dcaa7fe3

**Pull request merged**